src/Marten.Testing/Events/Projections/QuestMonsters.cs

```diff
@@ -13,150 +13,88 @@ public class Root
     public interface IMonstersView
     {
         Guid Id { get; }
-        string[] Monsters { get; }
+        List<string> Monsters { get; }
```

Comment:
STJ doesn't support arrays?

Reply:
It wasn't supporting it properly when behind the Array was another type. It's fixed in the System.Text.Json 5, so I reverted the change here with the dependency added.